editor/inspector_dock.cpp

```diff
@@ -211,15 +211,10 @@ void InspectorDock::_new_resource() {
 	new_resource_dialog->popup_create(true);
 }
 
-void InspectorDock::_load_resource(const String &p_type) {
-	load_resource_dialog->set_file_mode(EditorFileDialog::FILE_MODE_OPEN_FILE);
-
-	List<String> extensions;
-	ResourceLoader::get_recognized_extensions_for_type(p_type, &extensions);
-
+void InspectorDock::_load_resources_ext(const List<String> &p_extensions) {
```

Comment:
I dont quite understand what this is for. Can we do without it?

Reply:
I did it to allow Loading multiple file extensions by using `Load Shader` option. Since Shader and ShaderInclude are different kind of resources they need to be supported by the file dialog.

Reply:
I see, maybe its probably better to do a custom dialog for this then

Reply:
I can revert this change if you don't want it and add a separate menu option... Creating a whole new dialog just for this... I don't know.. - maybe in the other PR?